site/content/docs/contributing/1.0-roadmap.md

```diff
@@ -15,25 +15,41 @@ as a project.
 
 High level goals for kind include:
 
-- [ ] Reach [1.0] status / 
-stability by the end of Q1 2019 / the Kubernetes 1.14 release cycle
+## Beta Requirements
+
+To reach "beta" [grade][deprecation-policy] kind needs to at minimum:
+
+- [x] Improve documentation (Though this will eternally be "In Progress" !)
+  - [X] create a documentation site - [#268]
+  - [ ] expand examples of using kind
+  - [x] cover known issues, debugging, work-arounds, etc.
+- [x] Reliably [pass][kind-conformance-dashboard] the Kubernetes [conformance tests]
+ - [x] [Certify][certified] Conformance
 - [x] Support multi-node clusters - [#117]
-- [ ] Support offline / air-gapped clusters
-  - [ ] pre-loaded / offline CNI - [#200]
-- [ ] Support mounting host directories - [#62]
-- [ ] Improve Windows support
+- [x] Support offline / air-gapped clusters
+  - [x] pre-loaded / offline CNI - [#200]
+- [x] Support mounting host directories - [#62]
+- [x] Improve Windows support
   - [x] add Windows binaries to releases - [#155]
-  - [ ] improve instructions for KUBECONFIG in particular
-- [ ] Improve documentation - In Progress ...
-  - [ ] expand examples of using kind
-  - [ ] cover known issues, debugging, work-arounds, etc.
-  - [X] create a documentation site - [#268]
-- [ ] Improving logging and debuggability
-  - [ ] enable audit-logging
-- [ ] Support non-amd64 architectures (namely ARM) - [#166]
+  - [x] improve instructions for KUBECONFIG in particular
+- [ ] Support usage as a properly versioned, supported, and documented library 
+  - [ ] Improving logging and debuggability 
+for managing clusters in test harnesses
+- [ ] move API types / labels from `*.k8s.io` into [`*.x-k8s.io`][api-review-voluntary]
+- [x] Support all currently [upstream-supported Kubernetes versions][version-skew-policy]
+
+## GA Requirements
```

Comment:
I will add a target of code coverage, at least 70% per example

Reply:
Yeah, not sure what we're shooting for there, I would say 100% but I'd rather have a more informed idea first 😓 

The GA requirements in particular are incomplete which is why I added the last bullet 😅

Reply:
Nah, my experience says that above 70% is fine, is mostly just to avoid aggressive refactoring and regressions.
We can reword as "implement gating based on code coverage"

Reply:
this could be a beta requirement even, it needs more thought :-)

e2e coverage may be more important than unit even though we want unit, it's not obvious that unit test coverage will actually prevent eg incorrectly creating the containers.

Reply:
I´m a big fan of the Test pyramid concept https://martinfowler.com/articles/practical-test-pyramid.html, every testing layer has its importance. I think unit tests will be important once we start to add integrations like podman, ignite, and the public API moves to beta

Reply:
I don't think the unit tests will give use useful coverage of those backends. It will give us some coverage of the layers above. But in practice most of our bugs have been in the lower layers

Reply:
Please don't over-specify here, especially when we're not yet in beta.

A testing plan yes, that includes some level of unit and e2e sure, but the details of it can be spelled out after kind reaches beta.